docs/source/contribution.rst

```diff
@@ -93,9 +93,9 @@ Coding Guidelines
 
 We use `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ and a part of `OpenStack Style Guidelines <https://docs.openstack.org/developer/hacking/>`_ related to general coding style as our basic style guidelines.
```

Comment:
Could you fix this line?

Reply:
This is same as Chainer contribution guide.
https://docs.chainer.org/en/latest/contribution.html#coding-guidelines